ocaml/xapi/quicktest_cbt.ml

```diff
@@ -112,6 +110,42 @@ let vdi_data_destroy_test ~session_id ~vDI =
   | Test_failed msg -> failed test msg
   | e -> report_failure e test
 
+(* Check VDI.(copy, clone, and snapshot) all properly update cbt_enabled
```

Comment:
The comment should be updated: we're testing snapshot in the above test, not this one.